kumascript/macros/SpecData.json

```diff
@@ -489,6 +489,11 @@
     "url": "https://drafts.csswg.org/css-cascade-5/",
     "status": "WD"
   },
+  "CSS5 Colors": {
+    "name": "CSS Color Module Level&nbsp;5",
+    "url": "https://drafts.csswg.org/css-color-5/",
+    "status": "WD"
```

Comment:
This looks fine, except that the spec says it is an ED?

Reply:
all specs have an ED, and we seem to link to them rather than the TR version.

TR WD is here https://www.w3.org/TR/css-color-5/

Reply:
OK, fair point. I'll get this merged.